### DISCUSSION TITLE
Cannot update child in the Question set

### DISCUSSION THREAD

**Discussion by siddhi0719** [Sunbird-inQuiry/Community#97] [General Discussions]
@vinukumar-vs @rajeevsathish @rajeshkumaravel @Krishnaj20

I've created a Question set and which is currently live.

Following are the steps that I've followed to update the Live Question set:

Made changes in the Child section and was able to successfully sent it for review.
Tried to review the Question set from the Reviewer's workspace, not able to see the updated changes made in the child section.
I tried to check the network panel also, there is no API getting failed.

The above issue is been solved for 5.0 version we need to add the patch for 4.6 version.
Please let me know who can help me to add this patch.

**Comment by Krishnaj20** [comment]
@PoojaJManjooran @krgauraw - request you to look into this thread and respond.

**Reply by krgauraw** [reply]
Hi @vinukumar-vs @rajeevsathish, what is inquiry version used in ed 4.6 version?
cc: @PoojaJManjooran @Krishnaj20

**Comment by krgauraw** [comment]
Hi @siddhi0719 , Can you please share curl of the api which is getting called when data is getting loaded for reviewer. 
Also did you notice any difference in api call sequence between 4.6 and 5.0 version?

**Comment by siddhi0719** [comment]
Hi @krgauraw, 
1. This is the curl of the api when data is getting loaded for the reviewer

curl 'https://sunbirdsaas.com/content/composite/v1/search' \
  -H 'Accept: application/json' \
  -H 'Accept-Language: en-GB,en-US;q=0.9,en;q=0.8' \
  -H 'Connection: keep-alive' \
  -H 'Content-Type: application/json' \
  -H 'Cookie: connect.sid=s%3AteutRLRQyIPTeI9Sz-wjdhC2zqLgNjqE.anjmUJw5%2BT%2FSRWcIt0NgxescR2An%2F0KewvJXBxv3so0' \
  -H 'Origin: https://sunbirdsaas.com' \
  -H 'Referer: https://sunbirdsaas.com/workspace/content/upForReview/1' \
  -H 'Sec-Fetch-Dest: empty' \
  -H 'Sec-Fetch-Mode: cors' \
  -H 'Sec-Fetch-Site: same-origin' \
  -H 'User-Agent: Mozilla/5.0 (X11; Linux x86_64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/115.0.0.0 Safari/537.36' \
  -H 'X-App-Id: production.production.portal' \
  -H 'X-App-Version: 4.6.0' \
  -H 'X-Channel-Id: 0136351260261662723' \
  -H 'X-Device-ID: 87195a8edbba640e39096f3c7108bff9' \
  -H 'X-Org-code: 0136351260261662723' \
  -H 'X-Request-ID: 4dbf9d70-37af-e248-6a1c-7f5f751e37be' \
  -H 'X-Session-ID: teutRLRQyIPTeI9Sz-wjdhC2zqLgNjqE' \
  -H 'X-Source: web' \
  -H 'X-User-ID: 8934a4a7-f2ac-42cd-be07-b9f3bcfe81d6' \
  -H 'X-msgid: 4dbf9d70-37af-e248-6a1c-7f5f751e37be' \
  -H 'sec-ch-ua: "Not/A)Brand";v="99", "Google Chrome";v="115", "Chromium";v="115"' \
  -H 'sec-ch-ua-mobile: ?0' \
  -H 'sec-ch-ua-platform: "Linux"' \
  -H 'ts: 2023-10-10T16:02:11+05:30' \
  --data-raw $'{"request":{"filters":{"status":["Review"],"createdFor":["0136351260261662723"],"createdBy":{"\u0021=":"8934a4a7-f2ac-42cd-be07-b9f3bcfe81d6"},"objectType":"Content","primaryCategory":["Course Assessment","eTextbook","Explanation Content","Learning Resource","Practice Question Set","Teacher Resource","Exam Question","Content Playlist","Course","Digital Textbook","Question paper"]},"offset":0,"limit":9,"query":"","sort_by":{"lastUpdatedOn":"desc"}}}' \
  --compressed
 

2. We did not notice any difference in api call between 4.6 and 5.0 version
